app/client/components/organizationWrapper.jsx

```diff
@@ -0,0 +1,46 @@
+import React, {Component} from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import { updateActiveOrgRequest } from '../reducers/auth/actions'
+import { parseQuery, generateQueryString } from './../utils'
+import { withRouter } from "react-router";
+
+const OrganizationWrapper = Component => class extends Component {
+
+    componentDidMount() {
+        //load organisation if param is not matching currently active org
+        let query_params = parseQuery(location.search)
+        if(query_params["org"] && this.props.login.organisationId != query_params["org"]){
+            this.props.updateActiveOrgRequest(query_params["org_name"], query_params["org"])
```

Comment:
Why do we have `org_name` in the query string? I think it should just be pulled from `this.props.login.organisationName` so that there is a single source of truth. It would also simplify the query string url alot...

Reply:
changed! i initially added both params because I didn't realize the login object has an array of all the loaded organizations